### ISSUE TITLE
Places with place=town and capital=yes not rendering

### ISSUE THREAD

**Issue by martiensch** [gravitystorm/openstreetmap-carto#2399]
I noticed that the places Oranjestad and Philipsburg only started rendering after I added admin_level=3 to those towns. 
http://www.openstreetmap.org/node/50031810#map=12/12.5184/-70.0152
http://www.openstreetmap.org/node/50033577#map=12/12.0994/-68.8889

The same holds for Willemstad, but I upgraded it to city.
http://www.openstreetmap.org/node/50033577#map=12/12.0994/-68.8889

And someone else already changed Kralendijk with capital=yes -> capital=8, which also corrected the issue.
http://www.openstreetmap.org/node/50034650#map=12/12.1640/-68.2654

My guess is that the query in project.yaml for placenames-medium needs some fine-tuning in the [where clause](https://github.com/gravitystorm/openstreetmap-carto/blame/d0d4a81ffaa6372919460fd4dd61cd4a82597ee7/project.yaml#L1607)

**Comment by jojo4u** [comment]
I recently cleaned up capital=yes and wiki - [details on tagging.](https://lists.openstreetmap.org/pipermail/tagging/2016-October/030312.html)

What's important for osm-carto:
- there are no capital=2 remaining (there had been around 4), they have been converted to capital=yes
- the detection of capitals of countries should please ignore admin_level, imho this has semantic problems (see mailing list). There should be a "blacklist" of admin_level=3 and 4 and 6  because[ Russia is currently ](http://forum.openstreetmap.org/viewtopic.php?id=55959)not too keen to get their lower capitals changed (:/
- the first two examples from @martiensch will be converted to capital=3 by me

**Comment by dieterdreist** [comment]
2016-10-04 9:46 GMT+02:00 jojo4u <notifications@github.com>:

> the detection of capitals of countries should please ignore admin_level,
> imho this has semantic problems (see mailing list).



+1

**Comment by nebulon42** [comment]
Thanks for reporting, but I don't see a bug here, because `capital=yes` without any other additional tags should only be used for national capitals. The where clause you mentioned should make sure that everything that is handled by the capitals layer is excluded from placenames-medium.

Another question is the classification of national capitals. I'm open to the approach of @jojo4u, but I'm not convinced by the discussion on the tagging mailing list as there was not much response apart bikeshedding the discussion towards some situation in the Netherlands.

It would be interesting to hear the maintainers' opinion.

**Comment by jojo4u** [comment]
> Thanks for reporting, but I don't see a bug here, because capital=yes without any other additional tags should only be used for national capitals. 

The problem is that it's not rendered at all! For example Avarua is missing from the map. https://www.openstreetmap.org/node/266727954

![export](https://cloud.githubusercontent.com/assets/156724/19150939/1ead931c-8bca-11e6-8d51-5d10e7834d75.png)

**Comment by nebulon42** [comment]
Yes, you are right. That is a bug. The WHERE clause has to be rewritten anyway, so should read:
`AND (capital IS NULL OR capital != 'yes' OR (capital = 'yes' AND (admin_level IS NULL OR admin_level != '2')))` given that the criterion for country capitals remains unchanged.

Related to #2393.

**Comment by martiensch** [comment]
Hi Michael,

The new where clause looks fine to me ?

Cheers,

Martien

**Comment by martiensch** [comment]
(Sorry, I shall never again reply from my mail and insert emoji)

**Comment by sommerluk** [comment]
The WHERE clause looks good for me (but I haven’t tested it).

**Comment by nebulon42** [comment]
Since this will be closed once #2403 is merged I suggest that a separate issue for the discussion of country capital classification is opened (@jojo4u, @dieterdreist).

**Comment by nebulon42** [comment]
@pnorman this should have been closed when merging #2403. My fault, sorry.
